neutromeratio/ani.py

```diff
@@ -202,40 +207,45 @@ def __init__(self, alchemical_atoms=[]):
         super().__init__(alchemical_atoms)
 
 
+class Ensemble(torch.nn.ModuleList):
+    """Compute the average output of an ensemble of modules."""
+
+    def forward(self, species_input):
+        # type: (Tuple[torch.Tensor, torch.Tensor]) -> Tuple[torch.Tensor, torch.Tensor]
+        outputs = [x(species_input)[1].double() for x in self]
+        species, _ = species_input
+        energy = sum(outputs) / len(outputs)
+        return species, energy 
+      
+
+def load_model_ensemble(species, prefix, count):
```

Comment:
does this need to be defined, or inherited?

Reply:
I think we need to define this. This is originally a function in torchani/neurochem/__init__.py (https://github.com/aiqm/torchani/blob/master/torchani/neurochem/__init__.py) that loads the Ensemble() class from torchani.nn.Ensemble()  -- in our case this is the hook to get our local neutromeratio.ani.Ensemble() float64 class in.